3ds2/src/test/java/com/adyen/checkout/adyen3ds2/internal/ui/DefaultAdyen3DS2DelegateTest.kt

```diff
@@ -227,15 +231,8 @@ internal class DefaultAdyen3DS2DelegateTest(
 
         @Test
         fun `fingerprint is submitted automatically and result is completed, then details are emitted`() = runTest {
-            val authReqParams = TestAuthenticationRequestParameters(
-                deviceData = "deviceData",
-                sdkTransactionID = "sdkTransactionID",
-                sdkAppID = "sdkAppID",
-                sdkReferenceNumber = "sdkReferenceNumber",
-                sdkEphemeralPublicKey = "{}",
-                messageVersion = "messageVersion",
-            )
-            whenever(threeDS2Service.createTransaction(anyOrNull(), any())) doReturn TestTransaction(authReqParams)
```

Comment:
After updating the 3ds2 sdk this changed a bit. Maybe we should rebase to make sure we don't have big conflicts later